code-samples/eventing/writing-event-source-easy-way/index.js

```diff
@@ -49,6 +38,6 @@ function registerGracefulExit() {
 
 function logExit(message = 'Exiting...') {
   // Handle graceful exit
-  console.log(message);
+  process.stdout.write(`${message}\n`);
```

Comment:
what's wrong with using `consle.log` ?

Reply:
It's a small thing, but `console.log()` is asynchronous. Using `process.stdout.write()` ensures the message is actually written before exit.

Reply:
sounds good